app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    current_user.blank?
```

Comment:
tương tự câu hỏi như trên

Reply:
1. logged_in? Hàm này dùng để làm gì em.

Reply:
Kiểm tra user đã login hay chưa,Trả về true nếu user logged in, và false nếu không login

Reply:
current_user.blank? dòng này trả về true khi nào em

Reply:
dạ khi thằng user đó nil  thì trả về true